_descriptions/upload_description.md

```diff
@@ -1,21 +1,27 @@
-Thanks for connecting to *Open Humans*. You can now download your *23andMe* data. Below you can find a simple step-by-step guide to get access to the raw data file that we expect as input. If you have already uploaded data you can use the *File List* menu in the menu bar above to see the files you have already in your account.
+Thanks for connecting to *Open Humans*. Next, you can add your *Google Location* data. Below you can find a simple step-by-step guide to get access to the raw data file that we expect as input. If you have already uploaded data you can use the *File List* menu in the menu bar above to see the files you have already in your account.
 
-## How-to: Upload your Google Location History
+## How to add your Google Location History file
 
-<a href='static/steps.gif' target='_blank'><img src='static/steps.gif' width=100%/></a>
+### Get the data from Google
 
 - Go to <a href="https://takeout.google.com/" target="_blank">Google Takeout</a>
-- Select Location History (JSON)
+- Click "Select None"
+- Select "Location History (JSON format)"
 - Once your takeout file is compiled you can download it. You will also get an email once it's ready.
-- Download the Takeout file and re-upload it on this website.
-- You're done!
+- Download the Takeout file
+
+(Read more below to learn how to collect this data in the first place.)
+
+### Upload the file to this website
 
-(Read below to learn how to collect this data in the first place)
+Finally, **use the upload form on this webpage to add the file**. You're done!
+
+<a href='static/steps.gif' target='_blank'><img src='static/steps.gif' width=100%/></a>
 
-## How-to: Collecting your Google Location History
+## How to collect GPS with Google Location History
 You can enable the *Google Location History* from your iOS or Android phone you can use the *Google Maps* or *Google*  apps by turning on the following things.
 
-### *Google* app on iOS
+### *Google* app on Android
```

Comment:
Why android? The screenshots were made with iOS and i'm not sure whether one has to do any of these things on an android at all :)

Reply:
Oh, are these both instructions for iOS and no "Android"? I thought it was a typo.

If so, the above sentences should probably change to remove instructions about how to enable it for "Android"? Since there aren't actually instructions for that? (Is it just always on?)

Reply:
Right, both instructions are on how to turn it on, there's just two different iOS apps that you can use: `Google Maps` and `Google`. I don't have access to an android device, that's why there are no instructions so far. Maybe @mcescalante or @jhdulaney do have insights on that :)